app.js

```diff
@@ -2,7 +2,116 @@ const toggle = document.querySelector('.nav__links');
+    buttonText: 'See Project',
+  },
+  {
+    id: 3,
+    img: 'Image 3',
+    title: 'Multi-Post Stories Gain+Glory 3',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html', 'git'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus sed labore maxime neque, omnis at soluta eius quo non, dolorum amet a fuga aspernatur aliquam modi odit, tempore perspiciatis architecto! Assumenda nostrum veniam ab cupiditate, error quidem numquam voluptas? Consequatur alias repudiandae architecto beatae doloremque dolor sit, eveniet delectus corrupti.',
+    buttonText: 'See Project',
+  },
+  {
+    id: 4,
+    img: 'Image 4',
+    title: 'Multi-Post Stories Gain+Glory 4',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus sed labore maxime neque, omnis at soluta eius quo non, dolorum amet a fuga aspernatur aliquam modi odit, tempore perspiciatis architecto! Assumenda nostrum veniam ab cupiditate, error quidem numquam voluptas? Consequatur alias repudiandae architecto beatae doloremque dolor sit, eveniet delectus corrupti.',
+    buttonText: 'See Project',
+  },
+  {
+    id: 5,
+    img: 'Image 5',
+    title: 'Multi-Post Stories Gain+Glory 5',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus sed labore maxime neque, omnis at soluta eius quo non, dolorum amet a fuga aspernatur aliquam modi odit, tempore perspiciatis architecto! Assumenda nostrum veniam ab cupiditate, error quidem numquam voluptas? Consequatur alias repudiandae architecto beatae doloremque dolor sit, eveniet delectus corrupti.',
+    buttonText: 'See Project',
+  },
+  {
+    id: 6,
+    img: 'Image 6',
+    title: 'Multi-Post Stories Gain+Glory 6',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html', 'React'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus sed labore maxime neque, omnis at soluta eius quo non, dolorum amet a fuga aspernatur aliquam modi odit, tempore perspiciatis architecto! Assumenda nostrum veniam ab cupiditate, error quidem numquam voluptas? Consequatur alias repudiandae architecto beatae doloremque dolor sit, eveniet delectus corrupti.',
+    buttonText: 'See Project',
+  },
+];
+
+// Dynamic Content
+projectItems.forEach((item, index) => {
+  let techItems = '';
+  for (let i = 0; i < item.technologies.length; i += 1) {
+    techItems += `<li><a href="#" class="card__tags-links">${item.technologies[i]}</a></li>`;
+  }
+  const htmlContent = `
+    <div class="card">
+        <div class="card__img">${item.img}</div>
+          <h4 class="card__title">${item.title}</h4>
+            <ul class="card__tags">
+              ${techItems}
+            </ul>
+            <button class="btn" id=${index}>${item.buttonText}</button>
+    </div>
+`;
+  projectContainer.innerHTML += htmlContent;
+});
```

Comment:
Kindly modify the structure of the popup to match the design template.

Note the following:

- The description is the last item and justified
- The title of the project and the group of "See Live" and "See Source" buttons are on the same horizontal line and justifed
- The "See Live" and "See Source" each have icons
- The technologies used are left aligned
- There's more margin between the description and list of technologies
- There's a white overlay at the background of the detail popup

Your design:
![image](https://user-images.githubusercontent.com/20554583/216039475-309ee2dc-48bf-416d-af35-7be6390bddc3.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/216039586-6948c935-3201-481c-9b8b-ba380b72d116.png)